operator/internal/pkg/prepare/deploy.go

```diff
@@ -24,12 +25,18 @@ import (
 	"github.com/kyma-project/module-manager/operator/pkg/util"
 )
 
+type InstallCfgType string
+
 const (
-	configReadError = "reading install %s resulted in an error for " + v1alpha1.ManifestKind
+	InstallTypeSecret InstallCfgType = "local-secret"
```

Comment:
IMO, InstallCfgType is not related to what you are doing here, also its a strategy, not a type of installation.
Also as in my previous comment, there is no such thing as "remote-secret", so you can rename to "secret"